swagger.yml

```diff
@@ -575,8 +575,65 @@ paths:
           description: "You cannot like a comment twice"
           example:
             status: 409
-            message: "You cannot like a comment twice"
-          
+            message: "You cannot like a comment twice"   
+    /article/:page:
+      get:
+        tags:
+        - Articles
+        summary: "User can get all articles"
+        consumes:
+        - "application/json"
+        produces:
+        - "application/json"
+        parameters:
+        - in: "body"
+          name: "body"
```

Comment:
Good job all round. You have errors in line 589 and 590. The `in` should have a value of "path" and the `name` should have a value of "page". This is because it is a get request and has no response body, and also the params you are passing is clearly named 'page'. Please fix it. I noticed i made a similar mistake in my PR too.

Reply:
Thanks.